ftlib/ftlib_status.py

```diff
@@ -10,5 +8,8 @@ class FTAllReduceStatus(Enum):
     SUCCESS = 0
 
 
-class FTRebuildStatus(ConsensusStatus):
+class FTRebuildStatus(Enum):
```

Comment:
This change looks fine. But we need to modify the `ftlib.__init__.py` accordingly in another pr.

Reply:
Could you elaborate? `FTRebuildStatus` has all the enums that `ConsensusStatus` has so I think it should be fine. We can see how to improve that in a separate PR. This PR fixes the package installation issue since subclass from an enum class is not allowed.